python/graphstorm/eval/evaluator.py

```diff
@@ -1655,6 +1663,13 @@ def _get_major_score(self, score):
             major_score = score[self.major_etype]
         return major_score
 
+    def _get_early_stop_score(self, val_score):
+        """ By default, call _get_major_score
```

Comment:
This is an implementation detail, we can either rely on the docstring of the parent class, or include a functional description here (ie. what this function does, not how it does it)